### ISSUE TITLE
V4 analysis fail

### ISSUE THREAD

**Issue by deearahman** [artic-network/artic-ncov2019#77]
We have implemented the V4 into our pipeline. However, I am unable to perform the analysis. I am confused in terms of the V4 scheme and bed files. Any help would be great.

artic minion --normalise 200 --threads 4 --scheme-directory ~/artic-ncov2019/primer_schemes --fast5-directory ../fast5_pass --sequencing-summary ../sequencing_summary_FAQ69310_c642846d.txt --read-file POSITIVE_barcode23.fastq nCoV-2019/V4 POSITIVE
Scheme reference file not found: /home/stark/artic-ncov2019/primer_schemes/nCoV-2019/V4/nCoV-2019.reference.fasta

So basically I created a folder called V4 in the primer_schemes directory and placed all the V4 files in there. But the analysis won't run.
